Silence this warning:
  dreal/dreal/optimization/nlopt_optimizer.h:44:7: warning:
  'dreal::NloptOptimizer' declared with greater visibility than the type
  of its field 'dreal::NloptOptimizer::opt_' [-Wattributes]

TODO(jwnimmer-tri) Try to use a hidden inline namespace for dReal as well,
to silence this warning.

--- tools/dreal.bzl
+++ tools/dreal.bzl
@@ -25,6 +25,7 @@
     "-Woverloaded-virtual",
     "-Wpedantic",
     "-Wshadow",
+    "-Wno-attributes",
 ]
 
 # The CLANG_FLAGS will be enabled for all C++ rules in the project when
